helix-term/src/commands.rs

```diff
@@ -308,8 +308,7 @@ impl MappableCommand {
         goto_line_end, "Goto line end",
         goto_next_buffer, "Goto next buffer",
         goto_previous_buffer, "Goto previous buffer",
-        // TODO: different description ?
-        goto_line_end_newline, "Goto line end",
+        goto_line_end_newline, "Goto newline after line end",
```

Comment:
I actually prefer the current description: the line end is the newline character

Reply:
Okay, I can see why after looking in to the command.